libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/market-roles/market-role-group.service.ts

```diff
@@ -1,3 +1,20 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
issue: looks like this line was added before the license header, so another license header was added automatically in the CI as well. 

suggestion: remove the newly added license header and move the eslint comment bellow the existing license header.

suggestion: consider disabling an eslint rule per line, instead of for a whole file.

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1114/commits/4069d56e668a1cfeac17e111e7f1ab4d9d487175